src/lib/utils/ct_utils.h

```diff
@@ -416,6 +493,10 @@ class Mask final {
       */
       constexpr T value() const { return value_barrier<T>(m_mask); }
 
+      constexpr void const_time_poison() const { CT::poison(m_mask); }
+
+      constexpr void const_time_unpoison() const { CT::unpoison(m_mask); }
```

Comment:
Wait, this doesn't seem right. Are we poisoning/unpoisoning the masks currently somewhere?

Reply:
Yes. Here:

https://github.com/randombit/botan/blob/1af5545ae66ab7c56c35780769722db85217e982/src/lib/pk_pad/iso9796/iso9796.cpp#L192
https://github.com/randombit/botan/blob/1af5545ae66ab7c56c35780769722db85217e982/src/lib/tls/tls12/tls_cbc/tls_cbc.cpp#L452

Reply:
Ah ok, fallout from not having `CT::Choice` for this kind of logic.

Fine to merge this then, it's just keeping status quo. These should be fixed though.